assets/js/modules/search-console/components/dashboard/SearchFunnelWidgetGA4/ActivateAnalyticsCTA.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * ActivateAnalyticsCTA component for SearchFunnelWidgetGA4.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import VisitorsGraph from '../../../../../../svg/graphics/cta-graph-visitors.svg';
+import ConversionsGraph from '../../../../../../svg/graphics/cta-graph-goals.svg';
+import AnalyticsCTA from '../../../../../components/ActivateAnalyticsCTA';
+import PreviewGraph from '../../../../../components/PreviewGraph';
+
+export default function ActivateAnalyticsCTA() {
```

Comment:
This component is almost identical to the corresponding component in the `SearchFunnelWidget`. The only difference is the title of the goals/conversions preview. We can move it to the `common` folder and accept the title as a properly to make it available for both widgets.

Reply:
IMO, there is no point in making it reusable because we will delete the UA version once we have entirely switched over. So it will be simpler to have a separate one.